scripts/MirrorGarden-start-replica.sh

```diff
@@ -9,8 +9,9 @@ dfx start --background --clean &&\
 dfx canister create MirrorGarden &&\
 dfx build MirrorGarden &&\
 dfx canister install MirrorGarden
-ic-gt connect 127.0.0.1:8000 `dfx canister id MirrorGarden` --user '("Alice", (100, 200, 200))' &
-ic-gt connect 127.0.0.1:8000 `dfx canister id MirrorGarden` --user '("Bob", (200, 100, 200))' &
+
+# ic-mt connect 127.0.0.1:8000 `dfx canister id MirrorGarden` --user '("Alice", (100, 200, 200))'
```

Comment:
These are the lines that crash now, with this message:

```
Candid decoding error: Deserialize("There is an infinite loop in the record definition, the type is isomorphic to an empty type", "Trailing type: [I(4), U(1136381571), I(7), U(1269255460), I(10), I(4), U(1136381571), I(7), U(1269255460), I(10), I(4), U(1136381571), I(7), U(1269255460), I(10), I(4), U(1136381571), I(7), U(1269255460), I(10), I(4), U(1136381571), I(7), U(1269255460), I(10), I(4), U(1136381571), I(7), U(1269255460), I(10), I(4), U(1136381571), I(7), U(1269255460), I(10), I(4), U(1136381571), I(7), U(1269255460), I(10), I(4), U(1136381571), I(7), U(1269255460), I(10), I(4), U(1136381571), I(7), U(1269255460), I(10), I(4), U(1136381571), I(7), U(1269255460), I(10), I(4), U(1136381571), I(7), U(1269255460), I(10), I(4), U(1136381571), I(7), U(1269255460), I(10)]\nTrailing value: [19, 01, 03, 03, 00, 00, 00, 00, 00, 00, 01, 03, 03, 00, 03, 00, 00, 00, 00, 01, 03, 03, 00, 06, 00, 00, 00, 00, 01, 03, 03, 00, 09, 00, 78, 78, 00, 01, 03, 03, 00, 0c, 00, 00, 00, 00, 01, 03, 03, 03, 00, 00, 00, 00, 00, 01, 03, 03, 03, 03, 00, 00, 00, 00, 01, 03, 03, 03, 06, 00
```

Reply:
Can you rebuild candid with https://github.com/dfinity/candid/pull/155?

It adds the counter in the error message. If it's a concurrency bug, the counter would differ by a lot.

Reply:
Done.  It differs by one (17 versus 16).

Reply:
Notably, I also get the same numbers even when I run `ic-mt` with `-t`, which enables trace-level logging, and slows things down.  It doesn't seem likely that it's concurrency, based on that observation just now.

Reply:
I pushed the fix, can you try again?

Reply:
The fix worked!  Thanks!

![image](https://user-images.githubusercontent.com/1183963/103791954-4ef78b00-5000-11eb-89bf-39948733b0f6.png)


That's the graphics, as produced by `ic-mt`, on this PR now.